lang/src/org/partiql/lang/errors/ErrorCode.kt

```diff
@@ -500,6 +500,21 @@ enum class ErrorCode(private val category: ErrorCategory,
             "got ${errorContext?.get(Property.ACTUAL_ARGUMENT_TYPES)}"
     },
 
+    EVALUATOR_INVALID_PRECISION_FOR_TIME(
+        ErrorCategory.EVALUATOR,
+        LOCATION,
+        "invalid precision used for TIME type"),
+
+    /**
+     * This is a generic error wrapper for the DateTimeException thrown by Java's [java.time] when attempting to create
+     * an instance of [java.time.LocalTime] or [java.time.OffsetTime].
+     * The exception is caught by [org.partiql.lang.eval.time.Time.Companion.of].
+     */
+    EVALUATOR_DATETIME_EXCEPTION(
+        ErrorCategory.EVALUATOR,
+        LOCATION,
```

Comment:
Since the `DateTimeException` is wrapped around to throw this, it does throw an error with the meaningful message. 
For eg, `Time.of(12, 123, 12, 23, 3, 12)` throws the following error message. 
```
org.partiql.lang.eval.EvaluationException: Invalid value for MinuteOfHour (valid values 0 - 59): 123
```